requirements/test-mindeps.isolated

```diff
@@ -0,0 +1,23 @@
+# FIXME
+# this version of requests is newer than the minimum in package data
+# however, if we pin back to 2.19.1, we break 'responses' behaviors used by the
+# testsuite
+requests==2.22.0
+
+pyjwt==2.0.0
+cryptography==3.3.1
+typing_extensions==4.0
+
+## replicate the test dependencies here, rather than including the test.in file
```

Comment:
Is this comment correct? It looks like the `test.in` file is getting included at the bottom of the file.